bosh-dev/lib/bosh/dev/download_adapter.rb

```diff
@@ -33,9 +27,14 @@ def download_file(uri, write_path)
 
     def open_http_connection(proxy, uri, &block)
       tries = 0
+      http = nil
       begin
         http = Timeout.timeout(60) do
-          Net::HTTP.start(uri.host, uri.port, proxy.host, proxy.port, proxy.user, proxy.password)
+          http_opts = {read_timeout: 300}
+          if uri.scheme == 'https'
+            http_opts = http_opts.merge({use_ssl: true, verify_mode: OpenSSL::SSL::VERIFY_NONE})
```

Comment:
Did we try this without the disabled verification? Is there a reason for us to ignore the issues?